### PR TITLE
add progress bar and form validation

### PR DESCRIPTION
**Pull request by madelGeek**

### CODE REVIEW

File: app/scripts/controllers/modalBenefit.js [hunk 1]
```diff
@@ -15,10 +15,10 @@ angular.module('admissionSystemApp')
 // Please note that $modalInstance represents a modal window (instance) dependency.
 // It is not the same as the $modal service used above.
 
-  .controller('ModalInstanceBenefitCtrl', function ($scope, $modalInstance) {
+  .controller('ModalInstanceBenefitCtrl', function ($scope, $modalInstance, progressBarService) {
```
Comment (stas-demydiuk) [inline app/scripts/controllers/modalBenefit.js]: Controller's filename has to have the same name as controller, like MyCustomCtrl -> myCustom.js

Comment (madelGeek) [inline app/scripts/controllers/modalBenefit.js]: in modalBenefit.js I have ModalBenefitCtrl and ModalInstanceBenefitCtrl, both of which are responsible for modal window

File: app/scripts/controllers/progressBar.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+angular.module('admissionSystemApp')
+  .controller('ProgressBarCtrl', ['$scope', 'progressBarService', function ($scope, progressBarService) {
+
+
+    $scope.status = "Заповнено на: ";
```
Comment (stas-demydiuk) [inline app/scripts/controllers/progressBar.js]: Why do you extract this string to variable?

File: app/scripts/controllers/progressBar.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    $scope.value = progressBarService;
+
+    $scope.$watch('value.value', function (value) {
```
Comment (stas-demydiuk) [inline app/scripts/controllers/progressBar.js]: Function can be the first parameter of $watch expression, so, I think, there is no need to save progressBarService into $scope.value if we don't use it anywhere

**Pull request closed without merging**